ui-v2/app/components/consul-exposed-path-list/index.hbs

```diff
@@ -17,52 +17,52 @@
     <div>
       <ul>
       {{#if path.Protocol}}
-        <li class="protocol">
-          <span>
+        <dl class="protocol">
```

Comment:
I so love this PR! I'm gutted I spotted this - 😿  We should only have `li`'s within a `ul`.

Umm, is it possible to remove the `ul` now we are using `dl`s? The other option would be to put multiple `dt/dd`s inside one single `dl`. That can _sometimes_ cause problems, so I 'generally' opt for multiple `dl`s, but I think maybe we should just use a single `dl` here with multiple `dt/dd`s inside it.

Alternatively, maybe the easiest thing to do here is just remove the `ul`, lets chat about it later, I'm ok with either option.

Reply:
I'll try removing the `ul`.

Reply:
Everything works 💯 without the `ul`.